docs/user/filesystem.md

```diff
@@ -989,6 +989,10 @@ libpil4dfs intercepting summary for ops on DFS:
 ### Force pil4dfs related env set in child processes when calling execve and its variants
 Normally child processes inherit environmental variables from parent processes. In rare cases, e.g. scons, envs are striped off when calling execve(). It might be useful to force pil4dfs related env set in child processes by setting env "D_IL_ENFORCE_EXEC_ENV=1". This flag is 0 if not set.
 
+### Turn on compatible mode in libpil4dfs
+
```

Comment:
please add some comments on what is compatible mode and when/why it is needed.

Reply:
Thank you! I will add them.